bundler/lib/bundler/settings.rb

```diff
@@ -27,6 +27,7 @@ class Settings
       gem.changelog
       gem.coc
       gem.mit
+      gem.bundle
```

Comment:
Oh, good catch! This means we'll also need to document the setting in `## LIST OF AVAILABLE KEYS` in `bundler/lib/bundler/man/bundle-config.1.ronn`.

Reply:
Oh, I see you just skipped the setting in the quality spec, that's fine too 👍.